packages/vats/src/proposals/testUpgradedBoard.js

```diff
@@ -0,0 +1,35 @@
+import { E, Far } from '@endo/far';
+import { Stable } from '@agoric/internal/src/tokens.js';
+
```

Comment:
How about testing that an inventory with `getIDs()` looks the same before and after the upgrade?